README.md

```diff
@@ -143,32 +147,49 @@ defmodule YourApp.Router do
   use Phoenix.Router
 
   use BoomNotifier,
-    notification_trigger: [exponential: [limit: 100]]
+    count: [exponential: [limit: 100]]
     notifiers: [
       # ...
     ]
 ```
 
-### Notification trigger time limit
+### Time Based Throttle
+
+```elixir
+defmodule YourApp.Router do
+  use Phoenix.Router
+
+  use BoomNotifier,
+    notifiers: [
+        # ...
+      ],
+    groupping: :time,
+    throttle: :timer.minutes(1)
+end
+```
+
+### Time Limit
 
-If you've defined a triggering strategy which holds off notification delivering you can define a time limit value
-which will be used to deliver the notification after a time limit milliseconds have passed from the last error. The
-time counter is reset on new errors and only applies for cases where notifications are not sent.
+Both groupping strategies allow you to define a time limit that tells boom to deliver a notification if the amount
+of time has passed from the last time a notification was sent or error groupping started.
+
+If specified, a notification will be triggered even though the groupping strategy does not met the criteria. For
+time based throttling this is usefull if errors keep appearing before the throttle time and for count based throttle
+to reset (and notify) the grupping after a certain time period.
 
 ```elixir
 defmodule YourApp.Router do
   use Phoenix.Router
 
   use BoomNotifier,
-    notification_trigger: [:exponential],
+    count: [:exponential],
```

Comment:
Even though I like this name better I'd rather not to introduce api changes in this pull request

I think before releasing `v1.0.0` we should revisit the whole api and address the changes in a separate PR